authui/src/authflowv2/components/settings-user-profile-pic.css

```diff
@@ -0,0 +1,18 @@
+@layer components {
+  .settings-user-profile-pic__image-container {
+    @apply relative;
+    @apply h-22 w-22 tablet:h-50 tablet:w-50;
+  }
+
+  .settings-user-profile-pic__edit-button {
+    @apply inline-flex items-center justify-center;
+    @apply w-9 h-9;
+    @apply absolute bottom-0 -right-2 tablet:right-3.5;
```

Comment:
ltr trl

- This is stated in the guideline. https://github.com/authgear/authgear-server/issues/3626
- I want a stylelint rule to warn you guys automatically about this.